config/jira_economics.local.yaml.dist

```diff
@@ -33,7 +33,8 @@ parameters:
         footer: '<p style="margin: 0;">Venlig hilsen<br/> {{ sender.name }}</p>'
 
     gsorder_bundle:
-        field_debitor: customfield_10309
+        field_debitor: 'customfield_10309'
```

Comment:
Use 
https://github.com/aakb/jira_economics/blob/develop/src/Service/AbstractJiraService.php#L353
to get custom field ids.

Define them in jira_economics.local.yaml
https://github.com/aakb/jira_economics/blob/develop/config/jira_economics.local.yaml.dist#L3

Reply:
Fixed